File: Unidad 4/C#/U4programa4.cs
using System;

namespace U4programa4
{
    internal class Negativo
    {
        static void Main(string[] args)
        {

            int n = 0, num;
            Console.WriteLine("Ingrese los números a procesar: ");
            num = Convert.ToInt32(Console.ReadLine());

            int[] neg = new int[num];

            for (int i = 0; i < neg.Length; i++)
            {
                Console.WriteLine("Escribe un número en decimal negativo " + i + ": "); ;
                neg[i] = Convert.ToInt32(Console.ReadLine());

                if (neg[i] < 0)
                {
                    i++;
                }
                Console.WriteLine("Hay un total de " + i + " números negativos.");
            }

        }
    }
}

File: Unidad 4/Java/U4programa4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u4programa4;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class U4programa4 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in); 
        int n = 0, num;
            System.out.println("Ingrese los números a procesar: ");
            num = sc.nextInt();

            int[] neg = new int[num];

            for (int i = 0; i < neg.length; i++)
            {
                System.out.println("Escribe un número en decimal negativo " + i + ": "); ;
                neg[i] = sc.nextInt();

                if (neg[i] < 0)
                {
                    i++;
                }
                System.out.println("Hay un total de " + i + " números negativos.");
            }

        }
    }
